native/src/components/Header.tsx

```diff
@@ -145,7 +147,12 @@ const Header = (props: PropsType): ReactElement => {
     }
 
     const description = routeCityModel.prefix ? ` (${routeCityModel.prefix})` : ''
-    return `${routeCityModel.sortingName}${description}`
+    const cityNameLength = routeCityModel.sortingName.length
+    const minus = '-'
+    return cityNameLength < dimensions.deviceWidth / dimensions.headerTextSize &&
+      routeCityModel.sortingName.indexOf(minus) !== -1
```

Comment:
I think this check can be omitted, if no minus is in the string nothing is replaced.